runtime-decompiler/src/main/java/org/jrd/frontend/PluginMangerFrame/PluginConfigurationEditorController.java

```diff
@@ -63,6 +68,57 @@ public PluginConfigurationEditorController(PluginConfigurationEditorView view, P
         view.getOkCancelPanel().getCancelButton().addActionListener(actionEvent -> {
             view.dispose();
         });
+        view.getOkCancelPanel().getValidateButton().addActionListener(actionEvent -> {
+            if(view.getPluginListPanel().getWrapperJList().getSelectedIndex() == -1) return;
+            JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
+            //preparing data for the compiler
+            List<URL> dependencyURLs = ((DecompilerWrapperInformation)view.getPluginListPanel().getWrapperJList().getSelectedValue()).getDependencyURLs();
+            ArrayList<String> compileStringA = new ArrayList<>();
+            compileStringA.add("-cp");
+            StringBuilder dependencyS = new StringBuilder();
+            dependencyS.append("\"");
+            for (URL dependency: dependencyURLs)
+            {
+                dependencyS.append(":").append(dependency.getPath());
+            }
+            dependencyS.append("\"");
+            compileStringA.add(dependencyS.toString());
+            compileStringA.add("-d");
+            compileStringA.add("/tmp");
+            compileStringA.add(((DecompilerWrapperInformation) view.getPluginListPanel().getWrapperJList().getSelectedValue()).getWrapperURL().getPath());
+            System.out.println(compileStringA);
+            String[] compileString = compileStringA.toArray(new String[0]);
+
+            //compiling and getting error from the compiler
+            OutputStream errStream = new OutputStream() {
+                private String err = "";
+                @Override
+                public void write(int i){
+                    err += (char)i;
+                }
+
+                @Override
+                public String toString()
+                {
+                    return this.err;
+                }
+            };
+
+            int errLevel = compiler.run(null, null, errStream, compileString);
+
+            if (errLevel != 0) {
+                JOptionPane.showMessageDialog(view,
+                        "Validation failed: " + errStream.toString(),
```

Comment:
How long this can be? If it gores beyond some hundred of characters, then it would deserve better approach. But generally this shold work.
Maybe to print it all also to the logging botleneck?